opentracing/src/main/java/opentracing/Span.java

```diff
@@ -25,16 +25,7 @@
   /**
    * Suitable for serializing over the wire, etc.
    */
-  TraceContext traceContext();
-
-  /**
-   * Denotes the beginning of a subordinate unit of work.
-   *
-   * @param operationName name of the operation represened by the new span from the perspective of
-   * the current service.
-   * @return a new child Span in "started" state.
-   */
-  Span startChild(String operationName);
+  SpanContext childContext();
```

Comment:
> The `TraceContext traceContext();` method renamed to `SpanContext childContext();` so to avoid confusion as to whether this is to obtain the context for this object or the context for its children.

The original TraceContext class had two primary purposes:
  1. It itself serves as the span identity, meaning it encapsulates some form of trace-id and span-id and needs to be serialized on the wire.
  1. It was a container for Trace Attributes, or "distributed context", or 'baggage", which also needs to be serialized on the wire.

@michaelsembwever  From various comments on this thread, including the one quoted above, it seems your SpanContext is not meant to perform either of these functions.